.gitignore

```diff
@@ -10,6 +10,7 @@
 /bin/
 !/bin/install
 /composer.phar
+/composer.lock
```

Comment:
composer.lock is useful in an app. It should not be commited for libraries. :)

Reply:
Done